templates/chain.html

```diff
@@ -234,34 +234,74 @@ <h4>Manage operations</h4>
-                                      <select name="cleanup" id="queueCleanup" style="opacity:0.5" disabled="true">
-                                            <option value="1" selected>Clean up artifacts</option>
-                                            <option value="0">Turn off cleanup</option>
-                                      </select>
-                                      <select name="stealth" id="queueStealth" style="opacity:0.5" disabled="true">
-                                            <option value="0" selected>Run normal operation</option>
-                                            <option value="1">Run stealthy operation</option>
-                                      </select>
-                                      <button id="opBtn" type="button" style="opacity:0.5" disabled="true" class="button-success atomic-button" onclick="handleStartAction()">Start</button>
+                                          <table>
+                                              <tr>
+                                                  <td>
+                                                      <div id="control-box" class="control-box">
+                                                        <h1>Execution Control</h1>
+                                                        <hr>
+                                                        <table>
+                                                        <tr>
+                                                            <td>
+                                                                <button id="control-play" class="control-row-button" onclick="controlOp('run')" style="display: none;">
+                                                                   <span class="icon"><img src="/chain/img/play.png"></span>
+                                                                   <span class="caption">Resume Operation</span>
+                                                               </button>
+                                                                <button id="control-pause" class="control-row-button" onclick="controlOp('pause')">
+                                                                   <span class="icon"><img src="/chain/img/pause.png"></span>
+                                                                   <span class="caption">Pause Operation</span>
+                                                               </button>
+                                                            </td>
+                                                            <td>
+                                                                <button class="control-row-button" onclick="controlOp('cancel')">
+                                                                   <span class="icon"><img src="/chain/img/cancel.png"></span>
+                                                                   <span class="caption">Cancel Operation</span>
```

Comment:
I think we might want to tweak the Execution Control UI element... it seems a little out of place.  Maybe make the entire table part of the flex grid (inline-flex), set a min-width, and put the buttons above the PAUSED/RESUME/CANCELLED display box.